### DISCUSSION TITLE
Using the `ChildDatabase` and `ChildPage`

### DISCUSSION THREAD

**Discussion by muonmax** [jheddings/notional#64] [Q&A]
Can anyone give some guidance on how to use the `ChildDatabase` and `ChildPage` block types?

I am able to create databases programatically, but am unable to create an inline database. I have tried forking the repo and manually adding `is_inline` to the request in [`DatabasesEndpoint.create`](https://github.com/jheddings/notional/blob/9cc5d6a9a30920c2a26e5d1ee26eb305f445ca32/src/notional/session.py#L182-L191) without success.

Thanks.

**Comment by jheddings** [comment]
Hi @muonmax -

It looks like I need some work to support the `is_inline` property...  This needs to be set on the database directly, not the `ChildDatabase` block.  Unfortunately, Notional does not expose this property directly, but there is a workaround:

With a reference to the database, you can set the `is_inline` property directly using the following code:

```
notion = notional.connect(auth=auth_token)
notion.databases().update(database_id, is_inline=True)
```

The effect of calling the endpoint - `databases()` - exposes the underlying SDK.  This should work until I have the chance to update the Notional endpoints with the new features.  Note that you need the ID for the original database here.

Hope this helps!
